pabi_hr_expense_report/pabi_dunning_letter/wizard/pabi_dunning_letter_parser.py

```diff
@@ -3,6 +3,14 @@
 
 
 def pabi_dunning_letter_parser(cr, uid, ids, data, context):
+    if context.get('email_attachment', False):
```

Comment:
Hello Mustu, I do not quite understand this part of code. Why do we need data from context?  I though all data is already in data['parameters']
Also, given this, will email and print out being execute at the same time?